src/main/java/com/softserve/academy/spaced/repetition/service/MailService.java

```diff
@@ -63,4 +94,34 @@ private String getFreeMarkerTemplateContent(Map <String, Object> model) {
         }
         return "";
     }
+
+
+    private String getChangePasswordTemplateContent(Map <String, Object> model) {
+        StringBuilder content = new StringBuilder();
+        try {
+            content.append(FreeMarkerTemplateUtils.processTemplateIntoString(
+                    freemarkerConfiguration.getTemplate("changePasswordMailTemplate.txt"), model));
+            return content.toString();
+        } catch (IOException e) {
+            logger.log(e.getClass().getName());
+        } catch (TemplateException e) {
+            logger.log(e.getClass().getName());
```

Comment:
If exception handling is the same, use
`
} catch (IOException | TemplateException e) {`